charts/argo-controller/values.schema.json

```diff
@@ -79,6 +79,16 @@
         }
       },
       "required": ["podGarbageCollection", "workflowTTL"]
+    },
+    "prometheusMetrics": {
+      "type": "object",
+      "properties": {
+        "enabled": { "const": true },
```

Comment:
This will prevent ever disabling the metrics. If we want it to be that way, we can remove this value and hard-code `true`